### ISSUE TITLE
Error when using relative date query in metabase 0.34.0

### ISSUE THREAD

**Issue by skuroq** [dacort/metabase-athena-driver#26]
Hi,
first, thanks for the driver :) 

While testing out the new Metabase 0.34.0 Version i stumbled upon an error when doing relative date queries, for example when filtering 'today' on a date field.
I receive this error when using your provided jar from the release section:
![image](https://user-images.githubusercontent.com/19620164/71665367-ed517080-2d5c-11ea-9803-5857d5da6e03.png)

I tried rebuilding the athena driver from source. The resulting jar works fine when using metabase 0.33.7.3 but again i get an error in 0.34.0 but a different one this time:
![image](https://user-images.githubusercontent.com/19620164/71665429-31dd0c00-2d5d-11ea-82de-7e13ce8c2a62.png)

I quickly looked at the metabase source code and saw that in the release a lot a work was done concerning timezones and dates. Also the structure of the files changed, for example metabase/util/date.clj is now called metabase/util/date_2.clj. 
Maybe this has something to do with this?

**Comment by dacort** [comment]
Hi @skuroq - thanks for the issue report! There was indeed a lot of time-related work in 0.34 release, so it's likely something changed. I'll try to take a look sometime in the next few days and see if I can figure out what changed.

**Comment by dacort** [comment]
Yea, looks like that `unprepare-date-with-iso-8601-fn` got removed. The pre-existing Presto driver has some indication of how it was changed, so modeling that I'm able to get these type of queries working again.

For reference, here's a sample table for testing relative date queries:

```sql
CREATE TABLE default.mb_relative
WITH (
  format='PARQUET'
) AS
WITH source_data AS (
  SELECT 0 AS "id", 'damon' AS "name", localtimestamp AS "timestamp"
  UNION ALL
  SELECT 1, 'bob', localtimestamp - INTERVAL '2' DAY 
)

SELECT * FROM source_data
```

As mentioned in #27 I wanted to see if I can get some basic tests running in order to better catch/validate things like this in the future. I'll try to get a new release of this packaged up early next week.

**Comment by skuroq** [comment]
Thanks for looking into it @dacort !

**Comment by dacort** [comment]
As an FYI, I've managed to get integrated Metabase tests working in the `feature/integration_tests_v2` branch. There are some failures to resolve, though, so I probably won't let that block rolling out a fix for 0.34. I'll try to get that pushed out tomorrow.
